src/presentation/http/router/note.ts

```diff
@@ -21,6 +24,23 @@ interface NoteRouterOptions {
   noteSettingsService: NoteSettingsService,
 }
 
+/**
+ *
+ * @param note - note to change
+ * @returns note with only one id
+ */
+function changeNoteToNotePublic(note: Note): NotePublic {
```

Comment:
It looks like utility function, so move it to for example domain/utils/definePublicNote.ts or to the domain/entitieis/notePublic.ts

```suggestion
function definePublicNote(note: Note): NotePublic {}
```